components/domActions.js

```diff
@@ -0,0 +1,67 @@
+import createButton from "../utilities/createButton.js";
+import notifyUser from "../utilities/notification.js";
+import { createTodoDiv, textArea } from "./domElements.js";
+import { todoWrapper } from "./domElements.js";
+import { todos } from "../entry.js";
+
+export const openCreateTodo = () => {
+  createTodoDiv.classList.add("open-create-todo");
+  textArea.focus();
+};
+
+export const cancelTodo = () => {
+  createTodoDiv.classList.remove("open-create-todo");
+  textArea.value = "";
+};
+
+export const renderTodoList = () => {
+  todoWrapper.innerHTML = "";
+
+  if (todos) {
+    todos.forEach((todo) => {
+      todoWrapper.append(createTodoDomElement(todo));
+    });
+  }
+};
```

Comment:
We don't need the `if` check as we know that `todos` is always an array
```suggestion
export const renderTodoList = () => {
  todoWrapper.innerHTML = "";
  
  todos.forEach((todo) => {
      todoWrapper.append(createTodoDomElement(todo));
   });
};
```